clang/lib/CodeGen/CGDeclCXX.cpp

```diff
@@ -769,9 +777,10 @@ void CodeGenModule::EmitCXXModuleInitFunc(Module *Primary) {
       CharUnits GuardAlign = CharUnits::One();
       Guard->setAlignment(GuardAlign.getAsAlign());
       GuardAddr = ConstantAddress(Guard, Int8Ty, GuardAlign);
+      IsTLS = Guard->isThreadLocal();
```

Comment:
This is always just `false`.  And I believe that's correct, module initialization is global, not thread-local.